central/graphql/resolvers/generated.go

```diff
@@ -1320,6 +1320,7 @@ func registerGeneratedTypes(builder generator.SchemaBuilder) {
 		"skipTLSVerify: Boolean!",
 	}))
 	utils.Must(builder.AddType("Syslog", []string{
+		"extraFields: [KeyValuePair]!",
```

Comment:
How changes to this file are related to the rest of the PR?

Reply:
So, not entirely sure _why_ the change is happening, but its done by the code generation (i.e. `make go-generated-srcs`).

I'll try removing those and check whether CI is passing, otherwise the change has to be made (not _entirely_ sure why the CI isn't failing in the first place though).